packages/admin/src/targetGroups/addContacts/AddContactsGridSelect.gql.ts

```diff
@@ -11,9 +11,9 @@ const targetGroupBrevoContactsFragment = gql`
     }
 `;
 
-export const manuallyAssignedBrevoContactsGridQuery = gql`
-    query ManuallyAssignedBrevoContactsGrid($offset: Int, $limit: Int, $email: String, $targetGroupId: ID!) {
-        manuallyAssignedBrevoContacts(offset: $offset, limit: $limit, email: $email, targetGroupId: $targetGroupId) {
+export const assignedBrevoContactsGridQuery = gql`
+    query AssignedBrevoContactsGrid($offset: Int, $limit: Int, $email: String, $targetGroupId: ID!) {
```

Comment:
I would keep the `ManuallyAssignedBrevoContactsGrid` naming to easier differentiate between the 2 similar queries (all & manually)